torch/_inductor/utils.py

```diff
@@ -1153,6 +1163,11 @@ def is_big_gpu(index_or_device: Union[int, torch.device] = 0) -> bool:
     return True
 
 
+def is_big_gpu_xpu(device: torch.device) -> bool:
```

Comment:
We can combine the logic with CUDA as a single one. The only difference is the `min_sms` value. So that we could generalize the logic like `min_sms = 68 if gpu_type == 'cuda' else config.xpu.big_gpu_threshold`